PWGHF/D2H/Tasks/taskFlowCharmHadrons.cxx

```diff
@@ -135,7 +140,8 @@ struct HfTaskFlowCharmHadrons {
   /// \param cand is the candidate
   /// \param tracksQx is the X component of the Q vector for the tracks
   /// \param tracksQy is the Y component of the Q vector for the tracks
-  template <typename T1>
+  /// \param DeChannel is the decay channel
+  template <int DeChannel, typename T1>
```

Comment:
```suggestion
  template <DecayChannel channel, typename T1>
```

Reply:
Thanks for the comment.
It's a nice naming style.